multisig-wallet/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+name = "multisig-wallet"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+
+[dependencies]
+std = { git = "https://github.com/FuelLabs/sway-lib-std", branch = "master" }
```

Comment:
Delete to use implicit `std` that's matching the compiler version used. There's no guarantee `master`'s stdlib compiles with any release version of `forc`. Also regen lockfile.